site/docs/guides/admin/servers/configuring-an-open-lineage-server.md

```diff
@@ -61,6 +61,22 @@ For open lineage server following can be configured:
             "user": "admin",
             "password": "secret"
         },
+        "assetLineageTopicConnectionOverride": {
```

Comment:
This is not a valid connection object.  

It *must* have a connector type that indicates the type of connector to connect.  
It should have an endpoint to point to the location to connect to.